radixdlt-java/src/main/java/com/radixdlt/client/atommodel/tokens/UnallocatedTokensParticle.java

```diff
@@ -102,18 +100,14 @@ public RadixAddress getAddress() {
 		return this.tokenDefinitionReference.getAddress();
 	}
 
-	public RadixResourceIdentifer getTokDefRef() {
+	public RRI getTokDefRef() {
```

Comment:
Not really quite sure why we went with Tok instead of Token here.

Reply:
funner to say :)